src/99-appendices/01-contributors.md

```diff
@@ -5,7 +5,7 @@ Legend (source:
 
 | Emoji | Represents                                                                        |
 |-------|---------------------------------------------------------------------------------- |
-| 💬    | Answering Questions (on the mailing list, NeuroStars, GitHub, or in person, etc.) |
+| 💬    | Answering Questions (on the mailing list, NeuroStars, GitHub, or in person, and so on) |
```

Comment:
Pretty sure this is directly lifted off of a source. I just added this file to an ignore list.